testfuzz/README.md

```diff
@@ -26,4 +26,34 @@ BesuFuzz eof-container \
 
 There are pre-written Gradle targets for `fuzzEthereumJS`, `fuzzEvmone`,
 `fuzzGeth`, `fuzzNethermind`, and `fuzzReth`. Besu is always a fuzzing target.
-The `fuzzAll` target will fuzz all clients.
\ No newline at end of file
+The `fuzzAll` target will fuzz all clients.
+
+## p256verify
+
+Performs differential fuzzing of P256 signature verification between Java and Native (BoringSSL)
+implementations of the P256Verify precompiled contract. Tests all combinations of signature
+generation and verification to ensure consistency across implementations.
+
+### Prototypical CLI Usage:
+
+```shell
+BesuFuzz p256verify \
```

Comment:
cd `testfuzz/build/install/BesuFuzz` since it seems like you have to run this from that directory